### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Daniil-A4**
- [DEMO LINK](https://Daniil-A4.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Daniil-A4.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,3 +1,73 @@
...
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link,
+.is-active {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:46]: 3'rd point in checklist

File: src/index.html [hunk 2]
```diff
@@ -5,10 +5,51 @@
...
+              </li>
+              <li class="nav__item">
+                <a href="#samsung" class="nav__link">Samsung</a>
+              </li>
+              <li class="nav__item">
+                <a href="#smartphones" class="nav__link">Smartphones</a>
+              </li>
+              <li class="nav__item">
+                <a href="#laptopsComputers" class="nav__link"
+                data-qa="hover">Laptops & Computers</a>
```
Comment (denys-danyliuk) [inline src/index.html:34]: 13'th point in checklist

File: src/index.html [hunk 3]
```diff
@@ -5,10 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,300;0,700;0,900;1,100;1,200;1,300;1,400;1,500;1,800;1,900&family=Roboto:wght@100;500&display=swap" rel="stylesheet">
```
Comment (denys-danyliuk) [inline src/index.html:10]: do you need Montserrat font? 
you don't actually use it

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -5,10 +5,55 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container">
+        <div class="header__body">
```
Comment (Andriy-Shymkiv) [inline src/index.html:17]: don't understand why you need this wrappers? you can do the same with only header

File: src/index.html [hunk 5]
```diff
@@ -5,10 +5,55 @@
...
+                <a href="#samsung" class="nav__link">Samsung</a>
+              </li>
+              <li class="nav__item">
+                <a href="#smartphones" class="nav__link">Smartphones</a>
+              </li>
+              <li class="nav__item">
+                <a href="#laptopsComputers"
+                  class="nav__link"
+                  data-qa="hover"
+                >
```
Comment (Andriy-Shymkiv) [inline src/index.html:36]:
```suggestion
                <a 
                  href="#laptopsComputers"
                  class="nav__link"
                  data-qa="hover"
                >
```

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,74 @@
...
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link,
+.is-active {
```
Comment (Andriy-Shymkiv) [inline src/style.css:44]:
```suggestion
.nav__link {
```

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,70 @@
 body {
+  --blue-color: #00acdc;
   margin: 0;
+  font-family: "Roboto", sans-serif;
```
Comment (denys-danyliuk) [inline src/style.css:4]: 3'rd point in checklist

File: src/index.html [hunk 8]
```diff
@@ -5,10 +5,51 @@
...
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
```
Comment (denys-danyliuk) [inline src/index.html:26]:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
```
add spaces between multiline sibling blocks

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,70 @@
...
+  font-style: normal;
+  font-weight: 500;
+  line-height: 60px;
+  display: block;
+  color: black;
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
+  position: relative;
```
Comment (denys-danyliuk) [inline src/style.css:54]: should position change on hover?

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by denys-danyliuk — APPROVED** [review]

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**